util/uvmdvgen/env.sv.tpl

```diff
@@ -20,6 +20,10 @@ class ${name}_env extends dv_base_env #(
 % endfor
 % endif
 
+% if has_edn:
+  push_pull_agent#(.DeviceDataWidth(EDN_DATA_SIZE)) m_edn_pull_agent;
```

Comment:
Now I'm thinking whether it's better if we move these agent, scb, sequencer change to cip, and have has_edn knob in cip_base_env_cfg to decide to create it or not. 
Per https://docs.opentitan.org/hw/ip/edn/doc/, we have at least 6 IP interacting with EDN.
@sriyerg any thoughts?

Reply:
Agreed - lets keep this a bit more barebone.

Reply:
okay that sounds good too. I can change this PR to implement this suggestion.